DuckDuckGo/SmallOmniBarState.swift

```diff
@@ -30,25 +30,27 @@ struct SmallOmniBarState {
         let showShareButton: Bool = false
         let clearTextOnStart = true
         let allowsTrackersAnimation = false
-        let showSearchLoupe = !AppDependencyProvider.shared.voiceSearchHelper.isSpeechRecognizerAvailable
```

Comment:
I notice that sometimes this value uses `isSpeechRecognizerAvailable`, and in other places it uses `isVoiceSearchEnabled`. Intentional?

Reply:
Yes, the naming could be better but `isSpeechRecognizerAvailable` is about having the speechRecognizer feature on iOS itself available whereas `isVoiceSearchEnabled` is our feature, if the user turned it on/off.
i.e: `isVoiceSearchEnabled` will always be false if `isSpeechRecognizerAvailable` is false.

In some places (iirc) we have different behaviors depending on system settings or feature settings.

Maybe a better name would be:
* isSystemSpeechRecognizerAvailable
* isSFSpeechRecognizerAvailable

Feel free to change it.

Reply:
That's okay, I'm happy to keep the naming to be honest, just wanted to make sure this was intentional. :+1: